.evergreen/config.yml

```diff
@@ -605,8 +621,11 @@ functions:
   run-valid-ocsp-server:
     - command: shell.exec
       params:
+        shell: "bash"
         script: |
           cd ${DRIVERS_TOOLS}/.evergreen/ocsp
+          . ./activate-ocspvenv.sh
+
           ${PYTHON3_BINARY} -m venv ./venv
```

Comment:
What virtual environment were these OCSP tasks using before the inclusion of this `activate-ocspvenv` script?

Reply:
I think it uses the virtual environment from `./venv/${VENV_BIN_DIR|bin}/python`.